test/test.wpcom.js

```diff
@@ -20,4 +20,75 @@ describe( 'wpcom', function() {
 				.catch( done );
 		} );
 	} );
+
+	describe( 'wpcom.plans', function() {
+		describe( 'wpcom.plansList', function() {
+			it( '[1] should get available WordPress.com plans', function( done ) {
+				var wpcom = util.wpcomPublic();
+
+				wpcom.plansList( { apiVersion: '1' } )
+					.then( data => {
+						// testing object
+						assert.ok( data );
+						assert.ok( data instanceof Array );
+						done();
+					} )
+					.catch( done );
+			} );
+
+			it( '[1.2] should get available WordPress.com plans', function( done ) {
+				var wpcom = util.wpcomPublic();
+
+				wpcom.plansList( { apiVersion: '1.2' } )
+					.then( data => {
+						// testing object
+						assert.ok( data );
+						assert.ok( data instanceof Array );
+						done();
+					} )
+					.catch( done );
+			} );
+
+			it( '[1.3] should get available WordPress.com plans', function( done ) {
+				var wpcom = util.wpcomPublic();
+
+				wpcom.plansList( { apiVersion: '1.3' } )
```

Comment:
Just curious why we are testing 3 different versions of the API for this particular endpoint?